api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -20,5 +20,18 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
-
+<changeSet id="Trunk-5411:Add doseLimitUnits field to Drug" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="drug" columnName="dose_Limit_Units"/>				
+			</not>
+		</preConditions>
+		<comment>Adding column doseLimitUnits in table drug</comment>
+        <addColumn tableName="drug">
+            <column name="dose_Limit_Units" type="varchar(255)">
```

Comment:
Have you looked at the convention that we use for the rest of the table names?

Reply:
Have you also looked at the database type we use for concepts columns?

Reply:
no @dkayiwa i have looked at the conventions for table names here https://wiki.openmrs.org/display/docs/Conventions and its saying lower case singular form, also when you look through the different liquibases for the subsequent openmrs versions its what is being used

Reply:
As for the database type for concept columns i have opted to use Numeric datatype after having a look at https://wiki.openmrs.org/display/docs/Managing+Concept+Datatypes#ManagingConceptDatatypes-HelpfulNotes

Reply:
Can you show me just one existing liquibase changeset with that naming convention for column names?

Reply:
<changeSet id="20180808-passwordreset" author="harisu+fanyui">
		<preConditions onFail="MARK_RAN">
			<not>
				<columnExists tableName="users" columnName="email"/>				
			</not>
		</preConditions>
		<comment>Adding column users.email</comment>
        <addColumn tableName="users">
            <column name="email" type="varchar(255)">
            	<constraints unique="true"/>
            </column>
        </addColumn>
    </changeSet>
</changeSet>
	<changeSet id="TRUNK-4791-20180216" author="patrick">
		<preConditions onFail="MARK_RAN">
			<not>
				<columnExists tableName="provider" columnName="speciality_id"/>
			</not>
		</preConditions>
		<comment>Adding speciality_id column to provider table</comment>
		<addColumn tableName="provider">
			<column name="speciality_id" type="int" />
		</addColumn>
		<addForeignKeyConstraint baseTableName="provider"
								 baseColumnNames="speciality_id"
								 constraintName="provider_speciality_id_fk"
								 referencedTableName="concept"
								 referencedColumnNames="concept_id" />
	</changeSet>
@dkayiwa  i have gotten these two different changeSets for adding columns to tables users and provider respectively from liquibase-update-to-2.2.xml in the core

Reply:
<changeSet id="TRUNK-4791-20180216" author="patrick">
		<preConditions onFail="MARK_RAN">
			<not>
				<columnExists tableName="provider" columnName="speciality_id"/>
			</not>
		</preConditions>
		<comment>Adding speciality_id column to provider table</comment>
		<addColumn tableName="provider">
			<column name="speciality_id" type="int" />
		</addColumn>
		<addForeignKeyConstraint baseTableName="provider"
								 baseColumnNames="speciality_id"
								 constraintName="provider_speciality_id_fk"
								 referencedTableName="concept"
								 referencedColumnNames="concept_id" />
	</changeSet>

Reply:
Are they named the same was as your column?

Reply:
<changeSet id="TRUNK-4505-20180804" author="fruether">
		<preConditions onFail="MARK_RAN">
			<not>
				<columnExists tableName="person" columnName="cause_of_death_non_coded"/>
			</not>
		</preConditions>
		<comment>Adding cause_of_death_non_coded to the table person</comment>
		<addColumn tableName="person">
			<column name="cause_of_death_non_coded" type="varchar(255)" />
		</addColumn>
	</changeSet>
@dkayiwa  this changeSet is from the  liquibase-update-to-2.2.xml in the core

Reply:
Do you notice the difference between the convention for those names and yours?

Reply:
@dkayiwa i stand to be corrected if am going wrong here

Reply:
OK. Can you write those columns names here and also include yours?

Reply:
no sir, they are not named in the same way as mine column ..
@dkayiwa

Reply:
**the column names :**
<column name="fulfiller_comment" type="varchar(1024)" />
<column name="cause_of_death_non_coded" type="varchar(255)" />
<column name="speciality_id" type="int" />
**mine is:**

<column name="dose_Limit_Units" type="Numeric">
@dkayiwa these are the different column names in comparison with mine

Reply:
<column name="speciality_id" type="int" />
<column name="cause_of_death_non_coded" type="varchar(255)" />
<column name="dose_Limit_Units" type="Numeric">

Reply:
@dkayiwa  i have been trying to send the column names but they are not displaying here ..
Am seeing the difference in the changeset id and the datatype of the column